pkg/alertmanager/multitenant.go

```diff
@@ -129,19 +126,17 @@ func (cfg *MultitenantAlertmanagerConfig) RegisterFlags(f *flag.FlagSet) {
 }
 
 type multitenantAlertmanagerMetrics struct {
-	totalConfigs *prometheus.GaugeVec
+	invalidConfig *prometheus.GaugeVec
 }
 
 func newMultitenantAlertmanagerMetrics(reg prometheus.Registerer) *multitenantAlertmanagerMetrics {
 	m := &multitenantAlertmanagerMetrics{}
 
-	m.totalConfigs = promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
+	m.invalidConfig = promauto.With(reg).NewGaugeVec(prometheus.GaugeOpts{
 		Namespace: "cortex",
-		Name:      "alertmanager_configs",
-		Help:      "How many configs the multitenant alertmanager knows about.",
-	}, []string{"status"})
-	m.totalConfigs.WithLabelValues(configStatusInvalid).Set(0)
-	m.totalConfigs.WithLabelValues(configStatusValid).Set(0)
+		Name:      "alertmanager_invalid_config",
```

Comment:
As this is a bool, I think the name `alertmanager_config_invalid` would be better suited?

Reply:
done!